watergun_assassin/authentication/views.py

```diff
@@ -4,7 +4,9 @@
 from django.http import JsonResponse
 from django.views.decorators.csrf import csrf_exempt
 from django.contrib.auth.models import User
+from rest_framework.decorators import api_view 
 
+@api_view(['POST'])
 @csrf_exempt
```

Comment:
```suggestion
@permission_classes([AllowAny])
@csrf_exempt
```

Reply:
Import this decorator, and use it after any uses of the @api_view decorator. This basically says any user, authenticated or not, can hit this endpoint. We will restrict these later when I show you how our Authentication/Authorization flow works!